sys/asan.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 # ASAN="address leak memory undefined"
-ASAN=${ASAN:="address undefined signed-integer-overflow"}
+# ASAN="address signed-integer-overflow"  # Faster build
+ASAN=${ASAN:="address undefined"}
```

Comment:
why drop intovf?

Reply:
because `undefined` should also include `signed-integer-overflow` ... no harm in keeping it though (ff7198b16d7121212bf93a879baa5b517caf7f2e)